pkg/templates/externalccm/packet.go

```diff
@@ -34,9 +36,18 @@ import (
 )
 
 const (
-	packetImage          = "packethost/packet-ccm:v0.0.4"
+	packetImage          = "packethost/packet-ccm:v1.0.0"
 	packetSAName         = "cloud-controller-manager"
 	packetDeploymentName = "packet-cloud-controller-manager"
+
+	packetCloudSASecretName = "packet-cloud-config"
+	//nolint:gosec
+	packetCloudSASecretData = `
```

Comment:
Pretty sure that small map[string]string or even a struct with fields will do the trick without templates.

Reply:
I'm don't think `map[string]string` would work, but a struct with fields is a great idea.

Reply:
@kron4eg Fixed, PTAL.